Form/Type/SeoMetadataType.php

```diff
@@ -83,4 +95,28 @@ public function getName()
     {
         return 'seo_metadata';
     }
+
+    /**
+     * @param SeoMetadata $data
+     * @return null|SeoMetadataPersistenceModel
+     */
+    private function convertToPersisterModel(SeoMetadata $data)
+    {
+        if (!class_exists('Doctrine\Bundle\PHPCRBundle\DependencyInjection\Compiler\DoctrinePhpcrMappingsPass')
```

Comment:
I tried to inject `cmf_seo.backend_type_phpcr` and `cmf_seo.backend_type_orm` to recognize the persistence type, but that does noch work, as both parameters arn't available together. :-(
So this was my only chance to do it atm.